planet/clients/orders.py

```diff
@@ -332,11 +334,13 @@ async def download_order(
         LOGGER.info(
             f'downloading {len(locations)} assets from order {order_id}'
         )
+
```

Comment:
I'm still confused by all these extra lines 😆  This line, line 308 above and line 343 below.